docs/design/consistency.md

```diff
@@ -1,21 +1,19 @@
-# Integration
+# Consistency
```

Comment:
@sentientwaffle I know you wanted this doc to be kept as integration, but that seems like too vague a title for this one page and also it steals from other pages in the same category because they could also be considered integration. I had trouble trying to summarize it different but maybe there's a grouping for this page that makes sense to you but isn't so broad as "Integration" so we could keep the three sections together? Otherwise I think we should split them out as I've done.

Reply:
Splitting is find.

I probably combined them b/c I'm biased in favor is combining things into as few pages as possible. :sweat_smile: 
(I like single-page mega-docs ([e.g.](https://ziglang.org/documentation/master/)) because I think they are great for discoverability and creating a natural linear reading order, which can help with understanding when starting from scratch. Also they are easy to C-f to use as reference, since you don't have to remember where any particular info is located.)